force-app/main/default/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -165,21 +163,6 @@ public virtual with sharing class AFFL_MultiRecordTypeMapper {
         return labelsnames;
     }
 
-    @testVisible
```

Comment:
This is no longer used in the context of this change. These maps were all using the raw values from the affiliation mappings.